missing winerror attribute

```python
import errno
import sys

def error_says_socket_not_connected(error: OSError) -> bool:
    if sys.platform == 'win32':
        return (error.winerror == 10057)
    else:
        return (error.errno == errno.ENOTCONN)
```

```
$ mypy --platform=linux a.py
Success: no issues found in 1 source file
$ mypy --platform=win32 a.py
a.py:7: error: "OSError" has no attribute "winerror"  [attr-defined]
Found 1 error in 1 file (checked 1 source file)
```

--- a/stdlib/builtins.pyi
+++ b/stdlib/builtins.pyi
@@ -1297,7 +1297,8 @@ class OSError(Exception):
 
 EnvironmentError = OSError
 IOError = OSError
-WindowsError = OSError
+if sys.platform == "win32":
+    WindowsError = OSError
 
 class ArithmeticError(_StandardError): ...
 class AssertionError(_StandardError): ...

--- a/stdlib/builtins.pyi
+++ b/stdlib/builtins.pyi
@@ -1292,9 +1292,13 @@ class OSError(Exception):
     # filename, filename2 are actually Union[str, bytes, None]
     filename: Any
     filename2: Any
+    if sys.platform == "win32":
+        winerror: int
+
 
 EnvironmentError = OSError
 IOError = OSError
+WindowsError = OSError
 
 class ArithmeticError(_StandardError): ...
 class AssertionError(_StandardError): ...
@@ -1333,9 +1337,6 @@ class ModuleNotFoundError(ImportError): ...
 class IndexError(LookupError): ...
 class KeyError(LookupError): ...
 class UnboundLocalError(NameError): ...
-
-class WindowsError(OSError):
-    winerror: int
 
 class BlockingIOError(OSError):
     characters_written: int

--- a/stdlib/builtins.pyi
+++ b/stdlib/builtins.pyi
@@ -1297,8 +1297,7 @@ class OSError(Exception):
 
 EnvironmentError = OSError
 IOError = OSError
-if sys.platform == "win32":
-    WindowsError = OSError
+WindowsError = OSError
 
 class ArithmeticError(_StandardError): ...
 class AssertionError(_StandardError): ...

--- a/stdlib/builtins.pyi
+++ b/stdlib/builtins.pyi
@@ -1295,7 +1295,6 @@ class OSError(Exception):
     if sys.platform == "win32":
         winerror: int
 
-
 EnvironmentError = OSError
 IOError = OSError
 WindowsError = OSError

